getstarted.md

```diff
@@ -22,6 +22,70 @@ with Hadoop, as well as extensions to that foundation that make for totally
 awesome demos. You'll also find a few videos related to our work and of course,
 how to contact us!
 
+First, though, let's just take a look at how straightforward it is to get started with very complex software. You can do it during a lunch break:
```

Comment:
I disagree. Our "getting started" section currently doesn't give new people a non intimidating way to start. We just throw a bunch of charms at people, and hope that they'll swim instead of sinking.

I wanted to add something small and specific to the top of the page that would get people using our stuff, and get them curious to learn more.

It wouldn't be a bad idea to add a tutorials section with more tutorials. And we ultimately might end up replacing this particular tutorial with something different. But I think that it's important for our "getting started" page to actually get someone started ...